doc/reference/src/core-language.pdc

```diff
@@ -692,9 +694,62 @@ To-do: Lay out syntax-directed typechecker following McBride's paper.
 
 ## Examples
 
+### Dependent identity function
+
+The dependent identity function can be typed as: $⊢ λt.λx.(x \overset{1}{:} t) \overset{1}{:} (t \overset{0}{:} ∗_i) → (x \overset{1}{:} t) → t$.
+
+This should also typecheck if the $x$ argument usage is replaced with $w$ (instead of $1$).
+
+### SKI combinators
+
+#### S combinator
+
+The dependent S ("substitution") combinator can be typed as: $λt1.λt2.λt3.λx.λy.λz.x z (y z) \overset{1}{:} (x \overset{1}{:} ((a \overset{1}{:} t1) → (b \overset{1}{:} t2) → t3)) → (y \overset{1}{:} ((a \overset{1}{:} t1) → t2)) → (z \overset{2}{:} t1) → t3$.
+
+This should also typecheck if the $x$, $y$, and $z$ argument usages are replaced with $w$ (instead of $1$ and $2$).
+
+#### K combinator
+
+The dependent K ("constant") combinator can be typed as: $⊢ λt1.λt2.λx.λy.x \overset{1}{:} (t1 \overset{0}{:} ∗_i) → (t2 \overset{0}{:} ∗_i) → (x \overset{1}{:} t1) → (y \overset{0}{:} t2) → t1$.
+
+This should also typecheck if the $x$ and $y$ argument usages are replaced with $w$ (instead of $1$ and $0$).
+
+#### I combinator
+
+The dependent I ("identity") combinator can be typed as: $⊢ λt.λx.(x \overset{1}{:} t) \overset{1}{:} (t \overset{0}{:} ∗_i) → (x \overset{1}{:} t) → t$.
+
+This should also typecheck if the $x$ argument usage is replaced with $w$ (instead of $1$).
```

Comment:
[It did type checked with `Omega`](https://github.com/cryptiumlabs/juvix/pull/260/commits/35b5dc1364d1f3c408673704608ffab5175b474a)! Woohoo!